resources/views/menus/browse.blade.php

```diff
@@ -106,9 +106,14 @@
         });
 
         $('td').on('click', '.delete', function (e) {
-            id = $(e.target).data('id');
 
-            $('#delete_form')[0].action += '/' + id;
+            var form = $('#delete_form')[0];
+
+            var deleteFormAction = form.action;
+
+            form.action = deleteFormAction.match(/\/[0-9]+$/)
```

Comment:
I'm not sure how I feel about this solution.  It locks us into mandating numerical ID values.

Reply:
I didn't check on this issue, but... Can't be anything except `/`?
`/\/[^\/]+$/`

Reply:
That regex is essentially matching `/*` where `*` is 1 or more characters that cannot contain a `/`.  So it will match `/a`, `/---`, and `/abc123!@#`, but not `/`, `//`, `/diu/`, or `/abc/123/!@#`